db/schema.rb

```diff
@@ -0,0 +1,69 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+# rubocop:disable Metrics/BlockLength
+ActiveRecord::Schema[7.0].define(version: 20_220_810_124_313) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension 'plpgsql'
+
+  create_table 'comments', force: :cascade do |t|
+    t.integer 'authorid'
+    t.integer 'postid'
```

Comment:
- [ ] We need to keep our tables clean and remove unnecessary attributes. Kindly run new migrations to remove these `postid` and `authorid` from your comments table. You already have `post_id` and `author_id`, and those are enough.